package.json

```diff
@@ -2,6 +2,7 @@
   "private": true,
   "scripts": {
     "lint": "eslint --ext .js,.mjs,.ts,.vue .",
+    "linkinator": "linkinator dist/ --recurse --skip 'twitter.com|www.farmlands.co.nz|www.instagram.com'",
```

Comment:
https://github.com/centrapay/centrapay-docs/pull/783#discussion_r1342221022

Reply:
I have manually checked the links, check they match what Google search does so not sure what else we could do here.

Reply:
@MeganSteenkamp It'd be good to validate external links 👀 

If linkinator can't support this, it is worth considering alternatives

Reply:
@MeganSteenkamp what does recurse do? Is it possible that it is recursing on the external pages? Also would using a configuration file be useful? Looks like you can specify most options including `skip` https://www.npmjs.com/package/linkinator

Reply:
@MeganSteenkamp looks like you can specify verbosity to `debug` which might help you if you haven't done this already

Reply:
@tom-blake I've tried other tools and they all reproduce errors for the same links so I don't think it's to do with the tooling. 
Farmlands website ➡️ 0 (HTTP_UNDEFINED)
Twitter ➡️ 403 
Instagram ➡️ Failed only in the pipeline, I'll retry this.


@AndyClifford 
* Recurse performs recursion in the `dist` directory so it will go into the guides folder, and folders beneath that etc. 
* Yes I looked at debug option and it shows HTTP responses but didn't help do any additional debugging. 
* I had a config file but had feedback from @tom-blake that it would be nicer as a script to get rid of another config file 🧹